src/Data/Interpolation.hs

```diff
@@ -186,12 +187,12 @@ instance (ToTemplateValue a, ToJSON a) => ToJSON (Uninterpolated a) where
 maybeGen :: Gen a -> Gen (Maybe a)
 maybeGen x = frequency [(1, pure Nothing), (3, Just <$> x)]
 
-noEnv, noColons :: Gen T.Text
+noEnv, varNameAllowed :: Gen T.Text
 noEnv = fmap T.pack $ arbitrary `suchThat` (\ s -> not ("_env:" `isPrefixOf` s) && not (null s))
-noColons = fmap T.pack . listOf1 $ arbitrary `suchThat` (/= ':')
+varNameAllowed = fmap T.pack . listOf1 $ arbitrary `suchThat` (\c -> isAlphaNum c || c == '_')
```

Comment:
`suchThat` with such a narrow filter is likely to burn a lot of cycles generating and discarding non-matching chars. Something like `freq [(10, choose 'A' 'Z'), (5, choose 'a' 'z'), (2, choose '0' '9'), (1, pure '_')]`.

Reply:
Hmm. Actually `isAlphaNum` includes alpha _and_ numeric Unicode chars, so maybe not such a narrow filter after all. Probably fine as is.